specification/appendix/appendix-1-ha-ciphers.md

```diff
@@ -0,0 +1,101 @@
+## Appendix 1 - List of HA Ciphers
+
+Below is a list of all available HA Ciphers.  This list is definitive and will not change before NHAIS is retired
+
+This is sourced from the
+[NHAIS Developer Document Library](https://digital.nhs.uk/services/nhais/nhais-developer-document-library).
```

Comment:
Do you know which of the resources in that list. I wouldn't know where to begin.

Reply:
added additional link to actual appendix on that page